packages/kotti-ui/source/kotti-filter/types.ts

```diff
@@ -0,0 +1,171 @@
+import { Kotti } from '../types'
+import { SpecifyRequiredProps } from '../types/utilities'
+
+export namespace KottiFilter {
+	export enum FilterType {
+		SEARCH = 'SEARCH',
+		STRING = 'STRING',
+		NUMBER = 'NUMBER',
+		BOOLEAN = 'BOOLEAN',
+		DATE_RANGE = 'DATE_RANGE',
+		SINGLE_ENUM = 'SINGLE_ENUM',
+		MULTI_ENUM = 'MULTI_ENUM',
+	}
+
+	export namespace Operation {
+		export enum String {
+			CONTAINS = 'CONTAINS',
+		}
+
+		export enum Number {
+			EQUAL = 'EQUAL',
+			LESS_THAN = 'LESS_THAN',
+			LESS_THAN_OR_EQUAL = 'LESS_THAN_OR_EQUAL',
+			GREATER_THAN = 'GREATER_THAN',
+			GREATER_THAN_OR_EQUAL = 'GREATER_THAN_OR_EQUAL',
+		}
+
+		export enum Boolean {
+			EQUAL = 'EQUAL',
+		}
+
+		export enum DateRange {
+			IN = 'IN',
+		}
+
+		export enum SingleEnum {
+			EQUAL = 'EQUAL',
+		}
+
+		export enum MultiEnum {
+			ONE_OF = 'ONE_OF', // OR
+		}
+	}
+
+	export namespace OperationLabel {
```

Comment:
Not translated. Should use `KottiFilter.Translations` namespace like other translated components. You can probably even make it automatic by utilizing the `Record<A, B>` generic